docs/README.md

```diff
@@ -346,6 +354,7 @@ You can set plugin configuration using Jenkins [Configuration As Code](https://g
 ```yaml
 unclassified:
   awsCredentialsProvider:
+    cache: true  # cache is on by default
```

Comment:
Can you clarify if the cache is only for the list, or including the content of the secret as well? Thanks.

Reply:
Ok have updated the description a couple of lines above to be "Cache of credential names (on/off)". Does that make sufficient sense?